ruby/src/main/java/com/google/protobuf/jruby/RubyMessage.java

```diff
@@ -246,16 +246,15 @@ public IRubyObject methodMissing(ThreadContext context, IRubyObject[] args) {
     public IRubyObject dup(ThreadContext context) {
```

Comment:
dup was not actually duplicating repeatedFields.  see the test at https://github.com/google/protobuf/pull/332/files#diff-bbb59c88129c0bf6537d0487634b282dR764  If this is run on the master branch, it fails